app/controllers/admin/administrators/administrators_controller.rb

```diff
@@ -61,6 +61,17 @@ def update
         end
       end
 
+      def delete
+        authorize @administrator, :destroy?
+      end
```

Comment:
Hmm, not sure where best to handle the idea that permissions for `destroy` and `delete` are the same - whether it should be on controller level or on policy level. Thoughts?

Reply:
I'd be happy to have this on the ApplicationPolicy, and have it default to destroy, in the same way that `new` and `create` work

Reply:
Moved it to the application policy